readme.md

```diff
@@ -638,22 +569,9 @@ Recipients
 	// if you need this...
 	myRecipient.Card = new StripeCreditCardOptions()
 	{
-		// set this property if using a token
-		TokenId = *tokenId*,
-
-		// set these properties if passing full card details (do not
-		// set these properties if you set TokenId)
-		Number = "4242424242424242",
-		ExpirationYear = "2022",
-		ExpirationMonth = "10",
-		AddressCountry = "US",                // optional
-		AddressLine1 = "24 Beef Flank St",    // optional
-		AddressLine2 = "Apt 24",              // optional
-		AddressCity = "Biggie Smalls",        // optional
-		AddressState = "NC",                  // optional
-		AddressZip = "27617",                 // optional
-		Name = "Joe Meatballs",               // optional
-		Cvc = "1223"                          // optional
+		// if you are PCI compliant, you can use raw credit card data
+		// instead of a tokenID such as in [this example](https://github.com/jaymedavis/stripe.net#creating-a-token)
+		TokenId = *tokenId*
```

Comment:
TBH I totally misread this the first time around. I'd suggest a slightly wordier alternative like:

> A token identifier is acquired by exchanging credit card information through Stripe.JS or Checkout and is a symbolic representation of that card. See https://github.com/jaymedavis/stripe.net#creating-a-token for more details. If you're testing or are sure that you're otherwise fully PCI compliant, you can also consider passing raw credit card information as in <link to section on using raw CC>, but this is generally not recommended.